animation_url/src/routes/[slug]/+page.ts

```diff
@@ -92,24 +122,26 @@ const getPilot = async function (rigId: string): Promise<Pilot | undefined> {
   return { uri: trainer, type: MediaType.image };
 };
 
-const mediaTypes = new Map([
-  ["jpg", MediaType.image],
-  ["jpeg", MediaType.image],
-  ["png", MediaType.image],
-  ["svg", MediaType.image],
-  ["gif", MediaType.image],
-  ["mp4", MediaType.video],
-  ["mov", MediaType.video],
-  ["ogv", MediaType.video],
-  ["webm", MediaType.video],
-  ["3gp", MediaType.video],
-]);
-
-const getMediaType = function (uri: string): MediaType | undefined {
+const getMediaType = function (uri?: string): MediaType | undefined {
+  if (!uri) {
+    return undefined;
+  }
   if (uri.startsWith("data:image/")) {
     return MediaType.image;
   }
   const parts = uri.split(".");
   const extension = parts[parts.length - 1];
   return mediaTypes.get(extension);
 };
+
+const encodeSVG = function (uri?: string): string | undefined {
+  if (!uri) {
+    return undefined;
+  }
+  if (uri.startsWith("data:image/svg+xml;utf8,")) {
+    const parts = uri.split("data:image/svg+xml;utf8,");
+    return "data:image/svg+xml;base64," + btoa(parts[1]);
+  } else {
+    return uri;
+  }
+};
```

Comment:
some big svgs, eg, cryptopunks, are not base64 encoded and for whatever reason that combination makes them not render in an image tag. however, they will render when base64 encoded.